src/sentry/workflow_engine/endpoints/organization_workflow_index.py

```diff
@@ -19,6 +21,19 @@
 from sentry.workflow_engine.models import Workflow
 
 
+class OrganizationWorkflowEndpoint(OrganizationEndpoint):
+    def convert_args(self, request: Request, workflow_id, *args, **kwargs):
+        args, kwargs = super().convert_args(request, *args, **kwargs)
+        try:
+            kwargs["workflow"] = Workflow.objects.get(
+                organization=kwargs["organization"], id=workflow_id
+            )
+        except Workflow.DoesNotExist:
+            raise ResourceDoesNotExist
+
+        return args, kwargs
```

Comment:
Should this be in a shared place so we can use it in `OrganizationWorkflowDetailEndpoint`? (This looks like duplicate code from that endpoint. not sure if there's a better shared place that we can have this code).

Reply:
`endpoints/utils.py`? 🤷 can be a followup.